exercises/D-predicates/exercise.js

```diff
@@ -6,10 +6,10 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {return number<0}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {return number >0<10}
```

Comment:
Concise solution - very nice!
I had been using { return number > 0 && number < 10 } so it is good to see a shorter way of achieving the same outcome :)